apps/admin/backend/src/app.ts

```diff
@@ -49,12 +54,18 @@ function getCurrentElectionDefinition(
   return mostRecentlyCreatedElection?.electionDefinition;
 }
 
-function constructDippedSmartCardAuthMachineState(
+function constructAuthMachineState(
   workspace: Workspace
 ): DippedSmartCardAuthMachineState {
   const currentElectionDefinition = getCurrentElectionDefinition(workspace);
   return {
     electionHash: currentElectionDefinition?.electionHash,
+    // TODO: Pull jurisdiction from VxAdmin cert authority cert
```

Comment:
I'll tackle this and the other TODOs in this commit as I complete https://github.com/votingworks/vxsuite/issues/2911 and https://github.com/votingworks/vxsuite/issues/2912